docs/src/components/Layout/Header.tsx

```diff
@@ -21,8 +21,20 @@ import { useRouter } from 'next/router';
 import { Logo } from '@/components/Logo';
 import { FrameworkChooser } from './FrameworkChooser';
 import { SecondaryNav } from './SecondaryNav';
+import LinkButton from './LinkButton';
+import type { MouseEventHandler, ReactElement } from 'react';
```

Comment:
Nice, love the `import type`

Reply:
I love this too! Nice JS/TS distinguishment.